odoo/models.py

```diff
@@ -3047,6 +3047,21 @@ def get_metadata(self):
             r['noupdate'] = value.get('noupdate', False)
         return res
 
+    @api.multi
+    def get_base_url(self):
```

Comment:
Do we actually need that method when `website` is not installed?
If it is only used in `website`, I would simply add the method on model `base`.

Reply:
The method is used elsewhere